README.md

```diff
@@ -60,8 +60,26 @@ you can use `uucls` in the command line to manage the package.
 
 Initially, I created these classes for my personal use. I'm sharing them here in
 the hope that this perhaps saves some of you the work creating your own files.
-I'm very happy about PRs and/or comments. Ideally, raise an issue or just get in
-touch via my UU email.
+I'm very happy about PRs and/or comments, but you'll need to be OK with the
+CC0/public domain dedication (see below).  Ideally, raise an issue or just get
+in touch via my UU email.
+
+## (Un)License
+
+This software was originally written by Johannes Korbmacher
+([j.korbmacher@uu.nl](mailto:j.korbmacher@uu.nl)) and contains contributions
+from Ty Mees ([t.d.mees@uu.nl](mailto:t.d.mees@uu.nl)) and Kevin Quach.
```

Comment:
```suggestion
from Ty Mees ([t.d.mees@uu.nl](mailto:t.d.mees@uu.nl)) and Keven Quach.
```

Reply:
Uff, sorry for mispelling your name. I wasn't paying due dilligence!